src/controller/CHIPDeviceController_deprecated.h

```diff
@@ -79,6 +79,8 @@ class DLL_EXPORT ChipDeviceController : public Controller::DeviceStatusDelegate
 
     CHIP_ERROR SetUdpListenPort(uint16_t listenPort);
 
+    chip::app::CommandSender * GetCommandSender() { return mCommissioner.GetCommandSender(); }
```

Comment:
I wish we do not add more method to the deprecated version of `ChipDeviceController` but instead extend the new version.

Reply:
Then I made some change on this part and now ChipDevice has GetCommandSender() and SendCommand() api

I have to add GetDeviceController() in deperated controller, but I think this can also be a step by removing the old controller.

Reply:
agree, we should use CHipDevice instead of deprecated ChipDeviceController